packages/block-library/src/cover/style.scss

```diff
@@ -96,6 +96,19 @@
 		width: 100%;
 	}
 
+	// Prevents misplaced flex alignment in IE11.
+	&::after {
+		display: block;
+		content: "";
+		font-size: 0;
```

Comment:
That seems weird, could you include context for this fix in the CSS so future developers can understand this hack?

Reply:
Agreed. To be honest though, I'm not sure _why_ this works. 😄 

Maybe something like this would be more informative...

```	
// Using flexbox without an assigned height property breaks vertical center alignment in IE11. 
// Apphending an empty ::after element corrects this issue.
```

@webmandesign do you happen to know more details on the fix? There isn't much context [here](https://github.com/philipwalton/flexbugs/issues/231#issuecomment-362790042).

Reply:
Appending the pseudo element is sort of like providing a float clearing element; it tricks IE into giving the element an implicit height rather than collapsing to zero and preventing the vertical alignment.